### PR TITLE
Add a course for refactoring

### PR DESCRIPTION
**Pull request by malekzahedi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -726,7 +726,8 @@ Before you head over, read the [Contribution Guide](CONTRIBUTING.md) first. You
 
 - ### Refactoring
 
-  (In Progress)
+  - **Course**
```
Comment (lnxpy) [inline README.md:729]: Change "Course" title to "Video" please :)

**Review by lnxpy — CHANGES_REQUESTED** [review]
It looks good. Would you change the subsection title to "Video" please?
Much appreciate your love. <3

File: README.md (hunk 1 above)
Comment (malekzahedi) [reply README.md]: Done. Thanks for your comment.

**Review by bobycloud — APPROVED** [review]
Great, Thanks @lnxpy and @malekzahedi for this change.

**Pull request merged**